remoteworkitem/trackeritem_repository_test.go

```diff
@@ -0,0 +1,106 @@
+package remoteworkitem
+
+import (
+	"fmt"
+	"testing"
+
+	"golang.org/x/net/context"
+
+	"github.com/almighty/almighty-core/models"
+	"github.com/almighty/almighty-core/resource"
+	"github.com/almighty/almighty-core/test"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestConvert(t *testing.T) {
+	resource.Require(t, resource.Database)
+
+	// Setting up the dependent tracker query and tracker data in the Database
+	db.Exec(`DELETE FROM "tracker_items"`)
```

Comment:
There will be much weeping and gnashing of teeth if this ever gets excecuted against a production DB by accident ;-) Do you really need to empty this table before the test?